test/features/kogito_service_performance.feature

```diff
@@ -10,7 +10,7 @@ Feature: Kogito Service Performance
     Given Namespace is created
 
   @quarkus
-  Scenario Outline: Quarkus Kogito Service Performance without persistence
+  Scenario Outline: Quarkus Kogito Service Performance with requests <requests>, native <native> but without persistence
```

Comment:
Didn't know you can basically parameterize everything in the Scenario. It seems you can really use it everywhere :)

Reply:
Yeah it is with the new structure, they replace the placeholder(s) in title now also :)
Because each scenariooutline/example is now parsed as a pickle, whereas before it was still the feature structure